website/docs/reference/snapshot-configs.md

```diff
@@ -155,6 +150,17 @@ Configurations can be applied to snapshots using the [YAML syntax](/docs/build/s
 
 <ConfigGeneral />
 
+:::tip
+
+Configuring snapshots in YAML is recommended for new snapshot configurations. If you have existing snapshots using the `.sql` config, they'll continue to work as expected. When you're ready to migrate to YAML, we recommend:
+1. Creating a backup copy of your snapshots (or using version control)
+2. Converting the configurations one at a time, testing as you go
+3. Using `alter` statements as needed to ensure table consistency or using a script to apply the `alter` statements
```

Comment:
This shouldn't be needed if you're just moving from sql -> yml, right? Is this just if you _add_ one (or more) of the new configs?